contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -34,6 +34,11 @@ RETURNS text
 AS 'babelfishpg_tsql', 'tsql_get_functiondef'
 LANGUAGE C IMMUTABLE PARALLEL SAFE;
 
+CREATE OR REPLACE FUNCTION sys.tsql_get_expr(IN text_expr text DEFAULT NULL , IN function_id OID DEFAULT NULL)
```

Comment:
Also add this function to the upgrade script.

Reply:
Updated in next commit